test_cases/process_approval.js

```diff
@@ -0,0 +1,215 @@
+
+            cy.get('[data-cy="tab Order 499a4429-9cd3-43ea-bf5d-60e0caacc5eb 1169d7cf-d03d-4bd5-b282-4897b3329d7c"]')
+              .should('exist')
+              .click();
+
+            // Click on the button "New Order"
+
+            // cy.get('.webix_spin')
+            //   .should('not.be.visible');
+
+            cy.get('[data-cy="menu-item New Order ed0e35bb-9367-4fa8-83cc-b6318375fd56 25735600-076c-4f11-874b-d33863eea029"]')
+              .should('exist')
+              .click();
+
+            // On the field "test-kcs-process-stock", choose "Coke"
+
+            // cy.get('[view_id^="ABViewForm_"]')
+            //   .should('not.be.disabled');
+
+            cy.get('[data-cy="connectObject testkcsprocessstock 20027a41-14aa-4679-9f30-a066f153a89a 79d1f941-04c2-4d65-98e9-1303da4412c2"]')
+              .find('.webix_el_box')
+              .should('be.visible')
+              .click({ force: true });
+
+            // data-cy="connectObject options 9913a083-e62e-11ec-b7f9-02420a0000f0 20027a41-14aa-4679-9f30-a066f153a89a 79d1f941-04c2-4d65-98e9-1303da4412c2"
+
+            // cy.get('[data-cy^="connectObject options"]')
+            //   .should('be.visible');
+
+            cy.get('.webix_list_item')
+              .should('be.visible')
+              .contains('Coke')
+              .click({ force: true });
+
+            // On the field "amount", fill up "10"
+
+            cy.get('[data-cy="number amount 79902266-40e5-463a-9869-cee01ef955a0 79d1f941-04c2-4d65-98e9-1303da4412c2"]')
+              .type('10');
+
+            // Click on the button "save"
+
+            cy.get('[data-cy="button save 79d1f941-04c2-4d65-98e9-1303da4412c2"]')
+              .should('exist')
+              .click();
+
+            // Say "Approve" on the email
+
+            // cy.get('.webix_spin')
+            //   .should('not.be.visible');
+
+            // cy.get('[data-cy="ABViewGrid_b5ccd065-a499-48b0-ac86-6c0ae4247de2_datatable"]')
+            //   .contains('Coke');
+
+            cy.get('.webix_badge')
+              .should('exist')
+              .and('be.visible');
+
+            cy.get('[data-cy="inbox_icon"]')
+              .should('exist')
+              // .wait(5000)
+              .click({ force: true });
+
+            cy.get('.webix_accordionitem_header')
+              .should('exist')
+              .and('be.visible');
+
+            cy.get('[data-cy="inbox-accordion-app-holder-0ac51d6c-7c95-461c-aa8b-7da00afc4f48_b17f815f-cadc-4829-afb0-a7a726bde018"]')
+              // .should('exist')
+              .contains('Admin Verify')
+              .scrollIntoView()
+              .click();
+
+            cy.get('button[name="data"]')
+              // .should('exist')
+              .contains('Approve')
+              .scrollIntoView()
+              .click();
+
+            // cy.get('[data-cy="inbox_taskwindow_close"]')
+            //   .should('exist')
+            //   .click();
+
+            cy.get('.webix_el_box')
```

Comment:
what is this finding? we may want to see if there is a better way to do this.

Reply:
Sorry I still can't find the way to close the inbox. This one is work just for now.